keps/sig-testing/5468-invariant-testing/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: KEP Template
+kep-number: 5468
+authors:
+  - "@BenTheElder"
+owning-sig: sig-testing
+participating-sigs:
+  - sig-instrumentation
+  - sig-api-machinery
+status: provisional
+creation-date: 2025-09-04
+reviewers:
+  - "@aojea"
+  - "@pohly"
+  - "@richabanker"
+approvers:
+  - TBD
+
+see-also:
+  - "/keps/sig-api-machinery/4153-declarative-validation"
+
+# The target maturity stage in the current dev cycle for this KEP.
+# If the purpose of this KEP is to deprecate a user-visible feature
+# and a Deprecated feature gates are added, they should be deprecated|disabled|removed.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.35"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
```

Comment:
will update this pending above discussion about the alpha/beta/stable lifecycle not really making sense for this KEP https://github.com/kubernetes/enhancements/pull/5514/files#r2323831978